oshi-core/src/main/java/oshi/software/os/OSProcess.java

```diff
@@ -374,6 +374,23 @@
      */
     List<OSThread> getThreadDetails();
 
+    /**
+     * The number of minor (soft) faults the process has made which have not required
+     * loading a memory page from disk. Sometimes called reclaims. Linux only.
```

Comment:
Instead of "Linux Only" we'll need to explain Windows "everything's in minor" and no data on Solaris.

Reply:
sure. I will try and do the windows, and mac when we get the mac faults concept clear.